topi/include/topi/transform.h

```diff
@@ -563,5 +563,52 @@ inline Tensor take(const Tensor& a,
         }, name, tag);
 }
 
+/*!
+* \brief Crop the 2nd and 3rd dim of input tensor, with the corresponding size
+* of h_w or with width and height of the second input tensor.
+*
+* \param inputs The input tensors.
+* \param offset Crop offset coordinate.
+* \param h_w Crop height and width. Only used when there is only one input tensors.
+* \param center_crop Whether to center crop.
+* \param name The name of the operation.
+* \param tag The tag to mark the operation.
+*
+* \return Cropped tensor
```

Comment:
TVM/TOPI interface could be (Input Tensor, Array of Offsets, Array Of Lengths) to keep it more independent.

Reply:
Is this related to the doc string?